src/controller/auth.js

```diff
@@ -0,0 +1,19 @@
+const user = require('../models').user;
+
+module.exports = {
+  create(req, res) {
+    req.body = JSON.parse(JSON.stringify(req.body));
```

Comment:
You could incorporate validation before processing the user input

Reply:
Okay, I will consume the validation module one of the team members have created already.

Reply:
This comment has been here for four days, I was wondering why you have not resolved it.